core/app/models/spree/order.rb

```diff
@@ -179,6 +182,14 @@ def self.not_canceled
       where.not(state: 'canceled')
     end
 
+    def self.merged
```

Comment:
Is there a reason these aren't scopes?

Reply:
Only for consistency with the `::cancelled` and `::not_cancelled` class methods defined just above.